junit-jupiter-params/src/main/java/org/junit/jupiter/params/ParameterizedTest.java

```diff
@@ -207,4 +207,15 @@
 	 */
 	String name() default "{default_display_name}";
 
+	/**
+	 * If true, all arguments of the parameterized test implementing {@link AutoCloseable}
+	 * will be closed after {@code @AfterEach} methods and {@code AfterEachCallbacks}
+	 * have been called.
+	 *
+	 * @since 5.8
+	 * @see java.lang.AutoCloseable
+	 * @see ParameterizedTestParameterResolver
+	 */
```

Comment:
Please add the `@API` declaration.

Reply:
Do we put `@API` declarations on fields? The other ones don't have that :thinking: 
If so, would it be `STABLE`?

Reply:
> Do we put `@API` declarations on fields?

Only if such fields were part of the public API. For example, we would use `@API` for public constants.

However, `autoCloseArguments` is not technically a field. It's an annotation attribute which is technically a method in an interface, and it's a new part of the public API.

> If so, would it be `STABLE`?

We normally label new features like this `EXPERIMENTAL` until the next minor release (i.e., until it's been battle tested for a few months in the wild).

Reply:
I also think we should go with `EXPERIMENTAL` here.